web-src/src/helpers/NetworkHelper.js

```diff
@@ -11,6 +11,16 @@
  */
 import wretch from 'wretch';
 
+const getHeaders = () => {
+  const ENABLE_APPBUILDER_LOGGING_PARAM = 'appBuilderLogging';
```

Comment:
Nitpicking: I wouldn't reveal any internal details. `extraLogging` should be sufficient, or something similar and neutral.

Reply:
Also, make sure to test it.

Reply:
`extraLogging` sound good 💯

Reply:
@vtsaplin , I applied your suggestion and checked that the corresponding instruction from the README on how to set this parameter works.